pages/account/manage/link/[[...data]].js

```diff
@@ -75,7 +75,16 @@ export default function ManageLink({ BASE_URL, username, link }) {
     e.preventDefault();
 
     let method = "POST";
-    let putLink = { group, name, url, icon, isEnabled, isPinned, animation };
+    let selectedIcon = icon !== "" ? icon : "fa-globe";
```

Comment:
This should be `FaGlobe` because we use React icons 

```suggestion
    let selectedIcon = icon !== "" ? icon : "FaGlobe";
```

Reply:
okay! will change it